website/content/docs/commands/list.mdx

```diff
@@ -1,25 +1,54 @@
 ---
 layout: docs
-page_title: list - Command
+page_title: list - Vault CLI
 description: |-
-  The "list" command lists data from Vault at the given path. This can be used
+  List data from Vault at the given path. This can be used
   to list keys in a, given secrets engine.
 ---
 
 # list
 
-The `list` command lists data from Vault at the given path (wrapper command for
+List data from Vault at the given path. This can be used
+to list keys in a, given secrets engine.
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault list [options] PATH
```

Comment:
```suggestion
$ vault list [flags] <vault_path>
```

It doesn't look like the command supports any options, just the one argument and standard flags